hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/kafka/SqlAvroTest.java

```diff
@@ -338,6 +343,46 @@ public void test_nonInclusiveUnion() {
                 "Not in union [\"null\",\"boolean\",\"int\"]: " + Long.MAX_VALUE + " (Long) (field=info)");
     }
 
+    @Test
+    public void when_createAvroMapping_then_keyAndValueAreAvailableInInfoSchema() {
+        assumeFalse(useSchemaRegistry);
+        String name = createRandomTopic();
+        kafkaMapping(name, ID_SCHEMA, NAME_SCHEMA)
+                .fields("id BIGINT EXTERNAL NAME \"__key.id\"", "name VARCHAR")
+                .create();
+
+        String query = "SELECT * FROM information_schema.mappings";
+        try (SqlResult result = sqlService.execute(query)) {
+            SqlRow row = null;
+            int cnt = 0;
+            for (SqlRow r : result) {
+                ++cnt;
+                row = r;
+            }
+
+            assertEquals(1, cnt);
+            assertNotNull(row);
+
+            // Destructure the row: it must contain "catalog", "schema", "name", "external name", "type", "options"
+            assertEquals("hazelcast", row.getObject(0));
+            assertEquals("public", row.getObject(1));
+            assertEquals(name, row.getObject(2));
```

Comment:
these assertions test more infoschema in general than schema options. They are fine, but redundant here, as this should be covered elsewhere (is it?)

Reply:
They are covered as a part of `SqlInfoSchemaTest`. The first couple of schema assertions may seem redundant, but let them be for whole picture understanding.